drivers/hmis_csv_importer/app/models/hmis_csv_importer/hmis_csv_cleanup/force_project_enrollment_coc.rb

```diff
@@ -0,0 +1,77 @@
+###
+# Copyright 2016 - 2022 Green River Data Analysis, LLC
+#
+# License detail: https://github.com/greenriver/hmis-warehouse/blob/production/LICENSE.md
+###
+
+module HmisCsvImporter::HmisCsvCleanup
+  class ForceProjectEnrollmentCoc < Base
+    def cleanup!
+      enrollment_coc_batch = []
+
+      enrollment_coc_scope.find_each do |e_coc|
+        # Don't replace blanks
+        next if e_coc.CoCCode.blank?
+
+        # If we have a single CoC from the project, this will set it,
+        # if not, this will set it to nil
+        coc_code = coc_for(e_coc.ProjectID)
+        next if coc_code.blank?
+
```

Comment:
Maybe add to only touch ones that are wrong?
```ruby
next if coc_code == e_coc.CoCCode
```

Reply:
Good idea